seacrowd/sea_datasets/tgl_profanity/tgl_profanity.py

```diff
@@ -0,0 +1,115 @@
+import csv
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+from datasets.download.download_manager import DownloadManager
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Licenses, Tasks
+
+_CITATION = """
+@article{galinato-etal-2023-context,
+    title="Context-Based Profanity Detection and Censorship Using Bidirectional Encoder Representations from Transformers",
+    author="Galinato, Valfrid and Amores, Lawrence and Magsino, Gino Ben and Sumawang, David Rafael",
+    month="jan",
+    year="2023"
+    url="https://papers.ssrn.com/sol3/papers.cfm?abstract_id=4341604"
+}
+"""
+
+_LOCAL = False
+_LANGUAGES = ["tgl"]
+_DATASETNAME = "tgl_profanity"
+_DESCRIPTION = """\
+This dataset contains 13.8k Tagalog sentences containing profane words, together
+with binary labels denoting whether or not the sentence conveys profanity /
+abuse / hate speech. The data was scraped from Twitter using a Python library
+called SNScrape and annotated manually by a panel of native Filipino speakers.
+"""
+
+_HOMEPAGE = "https://huggingface.co/datasets/mginoben/tagalog-profanity-dataset/"
+_LICENSE = Licenses.UNKNOWN.value
+_SUPPORTED_TASKS = [Tasks.SENTIMENT_ANALYSIS]
+_SOURCE_VERSION = "1.0.0"
+_SEACROWD_VERSION = "1.0.0"
+_URLS = {
+    "train": "https://huggingface.co/datasets/mginoben/tagalog-profanity-dataset/resolve/main/train.csv",
+    "val": "https://huggingface.co/datasets/mginoben/tagalog-profanity-dataset/resolve/main/val.csv",
+}
+
+
+class TagalogProfanityDataset(datasets.GeneratorBasedBuilder):
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    SEACROWD_VERSION = datasets.Version(_SEACROWD_VERSION)
+
+    SEACROWD_SCHEMA_NAME = "text"
+
+    BUILDER_CONFIGS = [
+        SEACrowdConfig(
+            name=f"{_DATASETNAME}_source",
+            version=SOURCE_VERSION,
+            description=f"{_DATASETNAME} source schema",
+            schema="source",
+            subset_id=_DATASETNAME,
+        ),
+        SEACrowdConfig(
+            name=f"{_DATASETNAME}_seacrowd_{SEACROWD_SCHEMA_NAME}",
+            version=SEACROWD_VERSION,
+            description=f"{_DATASETNAME} SEACrowd schema",
+            schema=f"seacrowd_{SEACROWD_SCHEMA_NAME}",
+            subset_id=_DATASETNAME,
+        ),
+    ]
+
+    DEFAULT_CONFIG_NAME = f"{_DATASETNAME}_source"
+    CLASS_LABELS = ["1", "0"]
+
+    def _info(self) -> datasets.DatasetInfo:
+        if self.config.schema == "source":
+            features = datasets.Features(
+                {
+                    "text": datasets.Value("string"),
+                    "value": datasets.Value("string"),
+                }
+            )
+        elif self.config.schema == f"seacrowd_{self.SEACROWD_SCHEMA_NAME}":
+            features = schemas.text.features(label_names=self.CLASS_LABELS)
```

Comment:
since in `__init__.py` file of `schemas` dir uses alias of `from .text import features as text_features`, we can shorten this a bit
```suggestion
            features = schemas.text_features(label_names=self.CLASS_LABELS)
```